packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -752,7 +752,7 @@ export function LineItemSublabelAndPrice( { product }: { product: ResponseCartPr
 	const isDomainMapping = productSlug === 'domain_map';
 
 	if ( ( isDomainRegistration || isDomainMapping ) && product.months_per_bill_period === 12 ) {
-		const premiumLabel = product.extra?.premium ? translate( 'Premium' ) : null;
+		const premiumLabel = product.extra?.premium ? translate( 'Premium' ) : '';
```

Comment:
A bummer that we'll end up with an empty leading space in the case of an empty premium label. Anything we can/should separately fix, @sirbrillig?

Reply:
Yeah, I wouldn't worry about it right now. This has been here for a long time and I think it either doesn't ever happen or was designed to be empty.